platform/android/java/editor/src/main/java/org/godotengine/editor/BaseGodotEditor.kt

```diff
@@ -194,19 +191,14 @@ abstract class BaseGodotEditor : GodotActivity() {
 	}
 
 	@CallSuper
-	protected open fun updateCommandLineParams(args: List<String>) {
+	protected override fun updateCommandLineParams(args: List<String>) {
 		// Update the list of command line params with the new args
-		commandLineParams.clear()
-		if (args.isNotEmpty()) {
-			commandLineParams.addAll(args)
-		}
+		super.updateCommandLineParams(args)
 		if (BuildConfig.BUILD_TYPE == "dev") {
-			commandLineParams.add("--benchmark")
+			getCommandLine().add("--benchmark")
```

Comment:
You should create a `updatedArgs` variable:
- For dev builds, the variable combines `args` with the `--benchmark` parameter
- For non dev builds, `updatedArgs = args`

Then call `super.updateCommandLineParams(updatedArgs)`